Cargo.toml

```diff
@@ -35,4 +35,4 @@ compiletest_rs = "0.2.6"
 
 [workspace]
 members = ["src/librustc_mir"]
-exclude = ["xargo"]
+exclude = ["xargo", "cargo-miri-test"]
```

Comment:
The `members` line is probably redundant; everything below this folder is in the workspace automatically.